core/resource/service/resource_service.go

```diff
@@ -157,18 +157,18 @@ func (rs ResourceService) Deploy(ctx context.Context, tnnt tenant.Tenant, store
 			r.MarkValidationFailure()
 			continue
 		}
-		r.MarkValidationSuccess()
 
 		urn, err := rs.mgr.GetURN(r)
```

Comment:
provide the error information to the client as well

Reply:
Done